.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
     "image": "mcr.microsoft.com/devcontainers/typescript-node:20-bookworm",
-    "postCreateCommand": "npm install && npx playwright install-deps && npx playwright install",
+    "postCreateCommand": "npm install && npx playwright install-deps && npx playwright install && npm run dev",
```

Comment:
@tobyurff I played around with it a little more and [found a way to start the dev server in the background, using `nohup`](https://code.visualstudio.com/remote/advancedcontainers/start-processes#:~:text=In%20other%20cases%2C%20you%20may%20want%20to%20start%20up%20a%20process%20and%20leave%20it%20running.%20This%20can%20be%20accomplished%20by%20using%20nohup%20and%20putting%20the%20process%20into%20the%20background%20using%20%26.%20For%20example%3A).  So if we:

1. change the command to be `nohup bash -c 'npm run dev &'`
2. add `nohup.out` to the `.gitgnore`

then that works nicely 😎

Reply:
@tobyurff just an update on this.  It seems to be a bit flakey - if the Codespace disconnects, for instance, which can happen if it is left unattended, then http://localhost:3000 doesn't appear to be reachable.  I propose we still go with my proposed `nohup` background service approach above though, and see how it goes?

Reply:
@tobyurff another update on this.  The issue with port 3000 being occupied is a bit of a headache, as it then causes `npm start` to fail.  So I'm now leaning towards not trying to run the dev server automatically on Codespace startup, I think it's causing more headaches than the benefit, thoughts?

Reply:
I guess another option would be to start the dev server on a different port, so that there's no danger of `3000` port  already in use problems.  But I think that changing the port just adds to the cognitive load, for not a huge benefit?

Reply:
Here's another idea: if the main issue is that the terminal window takes too long to be "ready" when the Codespace loads, this can be solved simply by opening another terminal window, and starting the dev server there.  That works fine for me, try it out and if it works for you too then we can echo out a message on the `postCreateCommand` advising to do this - good plan?